tools/android_support/generate_pom_file.py

```diff
@@ -11,8 +11,7 @@
 THIS_DIR = os.path.abspath(os.path.dirname(__file__))
 
 # The template for the POM file.
-POM_FILE_CONTENT = '''
-<?xml version="1.0" encoding="UTF-8"?>
+POM_FILE_CONTENT = '''<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm fine landing this as is to unbreak things, but is there a nice way we could test this?

Reply:
One option is to unit test this code, but there isn't a good way to do that currently.  The other option is to consume the artifacts in Gradle, but I don't think Gradle is installed on LUCI.